relayer/src/setup/index.ts

```diff
@@ -1,8 +1,65 @@
-import { createTestAccounts } from "@lightprotocol/zk.js";
-import { getAnchorProvider } from "../utils/provider";
-
-export const testSetup = async () => {
-  const providerAnchor = await getAnchorProvider();
-  // TODO: use updated -- buildscript -> add relayer tests
-  await createTestAccounts(providerAnchor.connection);
-};
+import {
+  createTestAccounts,
+  initLookUpTable,
+  useWallet,
+  airdropSol,
+} from "@lightprotocol/zk.js";
+import {
+  getAnchorProvider,
+  getKeyPairFromEnv,
+  getRelayer,
+} from "../utils/provider";
+import { PublicKey } from "@solana/web3.js";
+import { readFileSync, writeFileSync } from "fs";
+import { AnchorProvider, BN } from "@coral-xyz/anchor";
+
+export async function relayerSetup() {
+  const anchorProvider = await getAnchorProvider();
+
+  await createTestAccounts(anchorProvider.connection);
+
+  let lookUpTable;
+  const path = "lookUpTable.txt";
+  try {
+    let lookUpTableRead = new PublicKey(readFileSync(path, "utf8"));
+    let lookUpTableInfoInit = await anchorProvider.connection.getAccountInfo(
```

Comment:
we should unify this with the config from cli, but we can do that in a different pr